CHANGELOG.md

```diff
@@ -17,9 +18,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - Renamed `PathSchedulerService` to `UnreferencedPathSchedulerService` to differentiate types.
 - Refactored internals of `beekeeper-cleanup` to support generic data Lifecycle deletions.
     - Refactored `PagingCleanupService` to be a generic orchestrator of Lifecycle handlers.
-
-## [1.2.1] - TBD
```

Comment:
Why has this been removed?

Reply:
I guess this was combined into a 2.0.0 release no 1.2.1 release made.